sproutsocial/seeds-packets
# [BCT-403] Update seedlings to allow usage of node 12

## Description:

* Currently you need to be using Node 8 in order to yarn install, this updates the seedlings node-sass dependency to allow you to use node 12.


## Jira:

- [BCT-403](https://sprout.atlassian.net/browse/BCT-403)


## How to validate

1) Check out branch
2) If you use NVM type `nvm use` otherwise make sure your `node -v` is 12+.
3) Run `yarn --force && yarn build`
4) See that it successfully builds
